<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetashellMainWindow</class>
 <widget class="QMainWindow" name="MetashellMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MetashellMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
    <item row="1" column="0" alignment="Qt::AlignTop">
     <widget class="QLabel" name="lbl_prompt">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string> &gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignTop">
     <widget class="QLineEdit" name="le_input">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QScrollArea" name="consoleScrollBar">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>694</width>
         <height>310</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
